test/Tester/SerializationTests/SerializationTests.ImmutableCollections.cs

```diff
@@ -19,7 +19,8 @@ void RoundTripCollectionSerializationTest<T>(IEnumerable<T> input)
             var output = SerializationManager.RoundTripSerializationForTesting(input);
             Assert.Equal<T>(input,output);
         }
-
+#if !NETSTANDARD_TODO
+    //blocked by Cannot use binary formatter as fallback serializer while running on .Net Core
```

Comment:
this is not due to binary formatter, but due to not loading the right serializers, and ends up trying to use the fallback serializer (which is binary formatter). Probably most of these are related to that... There should only be a few tests that use the binary formatter purposely